testsuite/features/reposync/srv_mgr_sync_list_products.feature

```diff
@@ -1,10 +1,10 @@
 # Copyright (c) 2018-2025 SUSE LLC
 # Licensed under the terms of the MIT license.
 
-Feature: List available products and enable them
+Feature: List available products
```

Comment:
Can we please consider to move this feature into the secondary parallelizable tests?
It does not seems to do any change on the product, and it does not make sense to have it as part of the reposync stage.